sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -240,3 +240,31 @@ export class Application extends RecordDataModel {
   )
   disbursementSchedules?: DisbursementSchedule[];
 }
+
+/**
+ * Interface for assessment payload
+ */
+export interface Assessment {
+  weeks: number;
+  totalFederalAward: string;
+  totalProvincialAward: string;
```

Comment:
As far as we know right now, are all these variables always present or we do know already some that could be nullable?

Reply:
The main reason we had the structure as 'any' for the assessment column is we were not sure about the variables that are present or not(nullable). Just wanted to confirm, if we have come to a conclusion on the variables list?

Reply:
@guru-aot Still we do not have conclusion about which one(s) can be nullable or not. We added interface to provide a signature to the data, even if more properties are added to the assessment payload, the added ones will be part of the payload in a dynamic way.

Reply:
An elaborated comment has been provided to this interface as discussed.